### PR TITLE
Remove fake server for NTLM tests

### PR DESCRIPTION
**Pull request by harryi3t**

### CODE REVIEW

File: test/integration/auth-methods/ntlm.test.js [hunk 1]
```diff
@@ -136,13 +78,13 @@ describe('NTLM', function () {
                     value: 'postman'
                 }, {
                     key: 'pass',
-                    value: 'password'
```
Comment (kamalaknn) [inline test/integration/auth-methods/ntlm.test.js]: Can you move username and password to the top of the file?

Comment (harryi3t) [reply test/integration/auth-methods/ntlm.test.js]: sure 👍

File: test/integration/auth-methods/ntlm.test.js [hunk 2]
```diff
@@ -29,62 +32,50 @@ describe('NTLM', function () {
...
-            ),
-            headers = {
-                initialized: {'www-authenticate': 'NTLM'},
-                type2message: {'www-authenticate': type2message}
-            };
+    describe('with in-correct endpoint', function () {
+        before(function (done) {
+            var clonedRunOptions = _.cloneDeep(runOptions);
 
-        ntlmServer = new http.Server();
```
Comment (kamalaknn) [inline test/integration/auth-methods/ntlm.test.js]:
```
var clonedRunOptions = _.mergeDeep({}, runOptions, { item: { request: 'http://postman-echo.com/get' }, environment: {...} })
```

File: test/integration/auth-methods/ntlm.test.js [hunk 3]
```diff
@@ -29,62 +32,50 @@ describe('NTLM', function () {
...
         });
     });
 
-    after(function () {
-        ntlmServer.close();
-    });
-
     describe('with in-correct details', function () {
         before(function (done) {
             runOptions.environment = {
```
Comment (kamalaknn) [inline test/integration/auth-methods/ntlm.test.js]: Same as above ☝️

File: test/integration/auth-methods/ntlm.test.js [hunk 4]
```diff
@@ -29,62 +32,50 @@ describe('NTLM', function () {
...
-                    password: password,
-                    domain: domain,
-                    workstation: workstation
-                }
-            ),
-            headers = {
-                initialized: {'www-authenticate': 'NTLM'},
-                type2message: {'www-authenticate': type2message}
-            };
+    describe('with in-correct endpoint', function () {
```
Comment (kamalaknn) [inline test/integration/auth-methods/ntlm.test.js]: `with request server not supporting NTLM`

**Pull request merged**